week07/jungsukwoo-crud-api/src/main/java/practice/jungsukwoocrudapi/member/MemberController.java

```diff
@@ -0,0 +1,41 @@
+package practice.jungsukwoocrudapi.member;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import practice.jungsukwoocrudapi.member.exception.MemberNotFoundException;
+
+@RestController
+@RequiredArgsConstructor
+public class MemberController {
+
+    private final MemberService memberService;
+
+    @PostMapping("/members")
+    public void signUp(@RequestBody CreateMemberRequest createMemberRequest) {
+        memberService.signUp(createMemberRequest);
+    }
+
+    @GetMapping("/members/{name}")
+    public MemberResponse getMember(@PathVariable String name){
```

Comment:
스프링 부트 3.2 부터 pathVariable의 파라미터를 명시해야 인식할 수 있다고 합니다.
그 외에도 기존 방법으로 추론하게 만드는 방법이 있지만 직접 명시하는 것을 권장합니다.
```suggestion
    public MemberResponse getMember(@PathVariable("name") String name){
```